app/views/posts/_form.html.erb

```diff
@@ -1,28 +1,17 @@
 <%= form_with(model: post, local: true) do |form| %>
-  <% if post.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(post.errors.count, "error") %> prohibited this post from being saved:</h2>
-
-      <ul>
-      <% post.errors.full_messages.each do |message| %>
-        <li><%= message %></li>
-      <% end %>
-      </ul>
-    </div>
-  <% end %>
 
   <div class="field">
-    <%= form.label :name %>
+    <%= form.label :Cook %>
```

Comment:
The symbol for the name only _happens_ to be the same as the text of the label.

The symbol should match the symbol used for the associated form field.

We can talk about how to update label text at a later date.

Reply:
Ok great I'll note this down for a later convo, thank you ❤️

Reply:
Do you want to change the symbols to match the form fields?